school/index.html

```diff
@@ -737,8 +734,6 @@ <h1 class="search_category">
 </div>
 
 
-<!--
-<section>
-  {# include search-form.html #}
-</section>
--->
+<script>
+  picc.pages.school();
```

Comment:
So there are other ways to do this to minimize having a global variable.

One is to use a page selector, and in the JS, check for that selector to run the code. So something like:

```js
if ($('.school-page').length) {
  school();
}
```
I'm pretty sure we did this on Peace Corps.

Another way is to split bundles up a little more. We did this on FEC, Josh would know more. Essentially, we had a main JS file where are the shared JS code was and then a js file for each page.

Again, I would say this is fine for now, but wanted to inform of what else can be done.